src/Pages/kiwanlee/Main/Main.js

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+
+import './Main.scss';
+import Nav from '../../../Components/Nav/Nav';
+//클래스형 표현
+
+class Main extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      noDisplaying: 'none',
+      displaying: 'inline-block',
+      comment: '',
+      commentsList: [],
+    };
```

Comment:
```suggestion
    this.state = {
      display: false
      comment: '',
      commentsList: [],
    };
```
display 라는 하나의 상태값으로 바꿔볼까요?